editor.planx.uk/src/pages/FlowEditor/lib/store/preview.ts

```diff
@@ -231,8 +231,8 @@ export const previewStore: StateCreator<
           if (passportValue.length > 0) {
             const existingValue = acc.data?.[key] ?? [];
 
-            const combined = existingValue
-              .concat(passportValue)
+            const combined = passportValue
+              .concat(existingValue)
```

Comment:
Both `prototype.array.concat(string)` and `prototype.array.concat(array)` are valid signatures.

Before, we were allowing for `any.concat(array)` which is what was causing the bug. `passportValue` is always an array so should always be what we concatenate onto.